lib/utils/execution-state-manager.js

```diff
@@ -177,13 +178,54 @@ class ExecutionStateManager {
     this._browserToModuleMap.set(browserId, browserModuleList);
   }
 
+  /**
+   * Add module metadata mapped by moduleName to moduleMetadata Map.
+   *
+   * @param {string} moduleName
+   * @param {number} total - Total number of tests
+   * @param {number} passed - Number of passed tests
+   * @param {number} failed - Number of failed tests
+   * @param {number} runtime - Runtime to execute tests in module
+   * @param {Array<string>} failedTests - A list of failed test names
+   */
+  _injectModuleMetadata(moduleName, total, passed, failed, runtime, failedTests) {
+    this._moduleMetadata.set(
+      moduleName,
+      {
+        moduleName,
+        total,
+        passed,
+        failed,
+        runtime,
```

Comment:
what is the unit of time here?
I would suggest naming this to `duration` or `executionTime` :)

Reply:
The duration is in ms. I just added the unit for duration inline comment and also in README.md

Reply:
It's okay for now, but in future maybe it could include 'Millis' on the end, or 'MS' or something like that, to make it obvious without having to read the comment or README? Generally names should be enough to describe what the symbol means.